pathfinding_service.rst

```diff
@@ -161,7 +166,8 @@ Example
             "path": ["0xalice", "0xeve", "0xdave", "0xbob"]
         },
         ...
-        ]
+        ],
+        "feedback_token": "aaabbbcccdddeeefff"
     }
```

Comment:
the feedback token counts now for the whole list of paths, right? How do we differentiate which path worked out well?

Reply:
All routes get stored in the PFS with the token, when the client answers we lookup the token for the request and see if it's still valid and check against the provided route.